lib/railway_ipc/models/published_message.rb

```diff
@@ -0,0 +1,12 @@
+module RailwayIpc
+  class PublishedMessage < ActiveRecord::Base
```

Comment:
Avoided added additional model validations till we do work in https://flatiron.atlassian.net/browse/MSB-57 if we need them.